src/framework/core/js/Fluid.js

```diff
@@ -1751,19 +1878,13 @@ var fluid = fluid || fluid_4_0_0; // eslint-disable-line no-redeclare
             event = fluid.makeEventFirer({
                 name: fluid.event.nameEvent(that, eventKey),
                 preventable: eventSpec === "preventable",
+                promise: eventSpec === "promise",
```

Comment:
Does this relate to those events which return a promise, and if so, I thought you were going to remove those. But I might have been mistaken.

Reply:
This it the implementation of latched events for https://issues.fluidproject.org/browse/FLUID-4883 - it's a new implementation, not one that is going to be removed.